docs/integrations/webhooks/overview.mdx

```diff
@@ -111,28 +76,7 @@ Below is an example of a webhook payload for a `user.created` event:
 }
 ```
```

Comment:
The payload should always be treated as unsafe until you validate the incoming webhook. Webhooks will originate from another server and be sent to your application as a POST request. A bad actor would fake a webhook event to try and gain access to your application or data.

Reply:
do you want me to add this bit to the webhook overview?

Reply:
Its probably worth it, yeah. For people new to Webhooks its super important to understand.